server/index.js

```diff
@@ -23,14 +24,19 @@ app.use('/test', (req, res) => {
   res.send('3-2-1 testing! Server is serving!');
 });
 
+// app.get('*', (req, res) => {
+//   res.sendFile('index.html', { root: __dirname })
+// })
+
 app.get('/:id', (req, res) => {
   if (req.params.id > 100 || req.params.id < 1) {
     let errorMessage = 'Out of range error! Please choose a number 1 - 100.'
     res.send(errorMessage);
+  // need to handle if id is in range ??
   } else {
-    res.sendFile('/index.html', (err) => {
+    res.sendFile('/Users/kimmybeee/Desktop/kimarie-bonuses/client/dist/index.html', (err) => {
```

Comment:
Does this work with `res.sendFile('../client/dist/index.html', ...)`? If so, you could avoid checking in unnecessary data about your computer?

Reply:
I will be changing that using path so it will be one less thing that I have to change when we deploy (where it will need a complete path.